tests/test_service.py

```diff
@@ -42,21 +42,6 @@ def test_loadplugin(server_type):
     assert loaded
 
 
-def test_launch_server_not_install():
```

Comment:
@PProfizi that should not be removed

Reply:
@cbellot000 added back under the name test_launch_server_full_path()